### DISCUSSION TITLE
How do I access a key from within a task's playbook?

### DISCUSSION THREAD

**Discussion by ZackaryWelch** [semaphoreui/semaphore#2085] [Q&A] [answered]
I feel like I'm missing something obvious here. If I create a key named "token", how do I access it within a playbook? The only way I can see right now is using the API to pull project keys.

**Comment by fiftin** [comment, marked as answer]
Hi @ZackaryWelch currently you can use keys only for cloning repos and for access to servers. 

I didn't find secure way to pass the keys to `ansible-playbook`.
